.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
What does this file do?

Reply:
It was created by firebase init earlier on. Have since separated dev folder from deployment folder.